### PR TITLE
Swipetweet

### PR DESCRIPTION
**Pull request by ghost**
Swipe when tweet
- removes tweet button from NewTweetPage and use swipe since more logic in sailfish UI 
- Page => Dialog
- Move "posttweet" logic to main.qml instead of NewTweetPage because page is destroyed onAccept
- rename internal => tweetInternal to not conflict with other pages since object is now global.
- Dont show New Tweet Pulley menu when pending tweet
- pageStack.push again if tweet fails.

Tested:

 reply, new and image (with normal twitter upload) is tested

### CODE REVIEW

**Comment by veskuh** [comment]
Cool stuff!

Couple of issues:
- DM is broken:
  qrc:/qml/tweetian-harmattan/main.qml:289: ReferenceError: tweetTextArea is not defined
- Since newtweetpage is now dialog it shouldn't use page header. Instead it should have dialog header and the header should not show title instead show acceptText which is action so text should be probably same as the button used to have. https://sailfishos.org//sailfish-silica/qml-sailfishsilica-dialogheader.html
- Minor thing, but updates to qmlapplicationviewer are just noise, since we do not use them.

File: qml/tweetian-harmattan/main.qml [hunk 1]
```diff
@@ -137,6 +139,160 @@ ApplicationWindow {
...
+                }
+                */
+            }
+            else if (tweetType == "RT") {
+                console.log("id" + tweetId)
+                Twitter.postRetweet(tweetId, postStatusOnSuccess, commonOnFailure)
+                pendingTweet = true
+            }
+            else if (tweetType == "DM") {
+                Twitter.postDirectMsg(tweetTextArea.text, screenName,
```
Comment (veskuh) [inline qml/tweetian-harmattan/main.qml]: I think this should be:
Twitter.postDirectMsg(tweetText, screenName,

and screenName needs to be passed here.

**Comment by ghost** [comment]
DM and Dialogheader fixed now

**Comment by veskuh** [comment]
Still issue with directMessage:
 unknown:289 - qrc:/qml/tweetian-harmattan/main.qml:289: ReferenceError: screenName is not defined

It needs to be passed to internalTweet object like other properties.

**Comment by ghost** [comment]
fixed

**Comment by veskuh** [comment]
Tack. Merged.

**Pull request closed without merging**